### PR TITLE
Server week 3

### PR DESCRIPTION
**Pull request by adamfweidman**
Pull Request for step 1, servlets

### CODE REVIEW

**Comment by adamfweidman** [comment]
PR for step 2

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -65,20 +66,23 @@ <h1>Adam's Personal Portfolio</h1>
 
       <p>Click <a href="random-pair.html"><button type="button">here</button>
       </a> to see a random pairing of Luna and Food!</p>
-    </div>
-    <div id="content">
```
Comment (tristan-jh) [inline portfolio/src/main/webapp/index.html:69]: No action required, just a general thought. It's often recommended to use <div> tags to wrap related content. This helps the structure of the html stay in sync with the logical structure of your page. eg: all gallery related content is under the same dom element. Div tags don't affect the layout of the page by default but you can use css to apply the effects you want, eg: setting margin-top to create visual space between sections in your doc.

I don't think you need to make these changes now, but something to keep in mind for more complex html projects.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -46,3 +46,9 @@ function randomFood() {
     foodHolder.innerHTML = '';
     foodHolder.appendChild(imgFood);
 }
+
+function getData() {
```
Comment (tristan-jh) [inline portfolio/src/main/webapp/script.js:50]: as with the above, each method should have a comment describing what it does.

**Review by tristan-jh — APPROVED** [review]
Looks good! Just one small comment to address.

**Comment by tristan-jh** [comment]
Oh, one more thing to add, since this is branched from the Luna-Session, make sure you merge the Luna-Session PR onto master before merging this CL. If you merge it now these changes will show up in Luna-Session which could confuse the review their.

**Review by bootsieh — APPROVED** [review]
+1 to Tristan's comments, once those are resolved: LGTM

**Pull request merged**